server/src/main/java/com/aws/greengrass/cli/CLIEventStreamAgent.java

```diff
@@ -556,11 +566,37 @@ public GetLocalDeploymentStatusResponse handleRequest(GetLocalDeploymentStatusRe
                     Topics deployment = localDeployments.findTopics(request.getDeploymentId());
                     DeploymentStatus status =
                             deploymentStatusFromString(Coerce.toString(deployment.find(DEPLOYMENT_STATUS_KEY_NAME)));
+                    LocalDeploymentStatus localDeploymentStatus = new LocalDeploymentStatus();
+                    localDeploymentStatus.setDeploymentId(request.getDeploymentId());
+                    localDeploymentStatus.setCreatedOn(
+                            new SimpleDateFormat().format(new Date(deployment.getModtime())));
```

Comment:
It's generally recommended to use DateTimeFormatter instead of SimpleDateFormat

Reply:
so this is actually when "deployment" topic is last modified -> when deployment last updated/finished instead of when deployment created right? not sure if "created on" is an accurate wording

Reply:
I had the same question during the demo section, do we expect a last modified timestamp or just when it got created initially ?

Reply:
Yes, this was my bad, the time stamp does get updated per each modification so fetching `modTime` will return last modified time and not created time. Modifying the changes to have deployment creation time.